4_arm.ipynb

```diff
@@ -606,6 +1096,19 @@
     "</div>"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "lang": "en"
+   },
+   "source": [
+    "<div class=\"alert alert-block alert-info\">\n",
+    "    <b>Task</b>\n",
+    "    <p>Spawn the can on the desk and write a code to pick the can.</p>\n",
+    "    <p>Use following command to spawn and delete the can.</p>\n",
```

Comment:
You described two operations here, so it feels like you are using at least two separate **commands**.  So I pluralized "command" to "commands".

    "    <p>Use the following commands to spawn and delete the can.</p>\n",